jekyll/_cci2/ecs-ecr.md

```diff
@@ -81,323 +81,57 @@ AWS_RESOURCE_NAME_PREFIX | Prefix for some required AWS resources. Should corres
 Every CircleCI project requires a configuration file called [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/).
 Follow the steps below to create a complete `config.yml` file.
 
-### Specify a Version
+### Build and Push the Docker image to AWS ECR
 
-Every `config.yml` starts with the [`version`]({{ site.baseurl }}/2.0/configuration-reference/#version) key.
-This key is used
-to issue warnings about breaking changes.
+The `build_and_push_image` job builds a Docker image from a Dockerfile in the default location (i.e. root of the checkout directory) and pushes it to the specified ECR repository.
```

Comment:
Would it be good to revise it to `The build_and_push_image job of the aws-ecr orb` and also link to the aws-ecr orb (https://circleci.com/orbs/registry/orb/circleci/aws-ecr)?

Reply:
What do you think of also mentioning somewhere near the beginning that the sample project makes use of orbs, or the aws-ecr and aws-ecs orbs specifically?

Reply:
Fixed